sympy/solvers/solveset.py

```diff
@@ -978,6 +985,330 @@ def _solveset(f, symbol, domain, _check=False):
     return result
 
 
+def make_expr_args(f):
```

Comment:
As of now the result of the routine will be used by `identifying helpers`.

Reply:
I suggest naming this `term_factors` since it returns the factors of all terms (contrary to what the current docstring says).

Reply:
I don't think it returns factors of the expression, it just gives the terms as it is present in the expression

```python
>>> make_expr_args(x**2 - 1)
[-1, x**2]
```

Reply:
Also will it be a good idea to have this method placed somewhere else, this could be used as a public function (I don't think sympy has anything of this sought).

Reply:
Oh sorry :), I miss understood a bit. I will update the name as well as the docstring